src/js/controller/controller.js

```diff
@@ -144,7 +145,8 @@ Object.assign(Controller.prototype, {
         _model.on('change:scrubbing', function(model, state) {
             if (state) {
                 _pause();
-            } else {
+                _stateOnScrubbing = _model.get('state');
+            } else if (_stateOnScrubbing === STATE_PLAYING) {
```

Comment:
This looks like a mistake even though it's not (updating `_stateOnScrubbing` right above the else).  See @karimMourra's comment https://github.com/jwplayer/jwplayer/pull/3283/files#r261038692

Reply:
I started looking into it, and it looks like we are changing the state (and firing `pause` event - checked in the latest develop). Without the pause, we can see the video frame changing while we are scrubbing.
Also checked `_model.get('state')` when we are done with scrubbing, and it returns `paused` independent of the state before scrubbing.